pkgs/applications/terminal-emulators/tabby/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchurl, pkgs, lib }:
+
+stdenv.mkDerivation rec {
+  name = "tabby";
+  version = "1.0.207";
+  src = fetchurl {
+    url = "https://github.com/Eugeny/tabby/releases/download/v${version}/tabby-${version}-linux-x64.deb";
+    hash = "sha256-PTTkL3+mYb7KM8fDUmgCuAF2lU88fYOstGWp/O5WZas=";
+  };
+
+  nativeBuildInputs = with pkgs; [
+    dpkg
+    makeWrapper
+  ];
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    dpkg -x $src .
+
+    runHook postUnpack
+  '';
+
+  buildInputs = with pkgs; [
+    glib
+    glibc
+    nss
+    nspr
+    atk
+    cups
+    dbus
+    libdrm
+    gtk3
+    pango
+    cairo
+    xorg.libX11
+    xorg.libXcomposite
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXrandr
+    libxkbcommon
+    mesa
+    expat
+    xorg.libxcb
+    alsaLib
+    libGL
+    libsecret
+    musl
+  ];
+
+  postInstall = ''
+    mkdir -p $out/{opt,bin}
+    cp -r opt $out
+    wrapProgram "$out/opt/Tabby/tabby" --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland --enable-features=WaylandWindowDecorations}}" --set LD_LIBRARY_PATH=${lib.makeLibraryPath buildInputs}
+    cp -r usr/* $out
+
+    cat > $out/bin/tabby << END
+    #!/bin/sh
```

Comment:
`#!/bin/sh` does not always exist. Use `#!/usr/bin/env sh` or `#!/usr/bin/env bash`, as Bash is pretty much required for anything in Nix to work anyway

Reply:
About the flag applied - I just looked at what the deb from Tabby did and copied the same thing. The Tabby deb does create a file in `/usr/bin` that is basically a wrapper to `/opt/Tabby/tabby` with the flag applied.

Idk what `--no-sandbox` does, but that's what the Tabby deb contains. I'm open to moving the `--no-sandbox` flag to be built-in to `/opt/Tabby/tabby` instead of in the file in `/bin` and Desktop file.

Reply:
In Nix we have different idiomatic patterns than whatever is used for the .deb — I think using a single wrapper and a simple symlink honestly works better